apps/scotty/scotty.go

```diff
@@ -171,11 +171,13 @@ func loadTestMdbChannel(count int) <-chan *mdb.Mdb {
 }
 
 func getMyHostName(machines []mdb.Machine, myIpAddrs []string) string {
+	myIpAddrsMap := make(map[string]bool, len(myIpAddrs))
```

Comment:
Since you're just using this as an existence map (a set), how about: map[string]struct{}

Reply:
Done.